Do not force the use of libc++ when compiling with Clang
Fix by David Denoncin
https://bugs.gentoo.org/729560

--- a/CMakeLists.txt	2020-08-09 14:04:40.095904996 -0000
+++ b/CMakeLists.txt	2020-08-09 14:04:59.449238208 -0000
@@ -43,10 +43,6 @@
  message (FATAL_ERROR "C++11 support missing. Try upgrading your C++ compiler. If you have a good reason for using an outdated compiler, please let us know at support@taskwarrior.org.")
 endif (_HAS_CXX11)
 
-if (${CMAKE_CXX_COMPILER_ID} STREQUAL "Clang")
-  set (_CXX11_FLAGS "${_CXX11_FLAGS} -stdlib=libc++")
-endif (${CMAKE_CXX_COMPILER_ID} STREQUAL "Clang")
-
 if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
   set (LINUX true)
 elseif (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
